index.yaml

```diff
@@ -371,12 +371,13 @@ entries:
       catalog.cattle.io/os: linux
       catalog.cattle.io/provides-gvr: longhorn.io/v1beta1
       catalog.cattle.io/release-name: longhorn
+      catalog.cattle.io/type: cluster-tool
```

Comment:
annotation was bumped as expected

Reply:
Why was it bumped now and not in previous PR that actually did the change ?

Reply:
@cbron , @brendarearden noted in https://github.com/rancher/charts/pull/1388 that she would be doing the cluster tool bump for backup in a separate PR.

Reply:
This one is for longhorn, there are ones here for monitoring too. Did we forget to regenerate the index in the other PR ?

Reply:
Usually `make charts` updates the index.yaml, I also have been regenerating the index.yaml, but maybe during a rebase I removed it. I went ahead and regenerated and push it up in its own commit.